portfolio/.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
I don't think that this file needs to be checked in. You might consider adding this file to a `.gitignore` file.